apps/web/pages/event-types/index.tsx

```diff
@@ -362,7 +371,9 @@ export const EventTypeList = ({ group, groupIndex, readOnly, types }: EventTypeL
       <ul ref={parent} className="divide-subtle !static w-full divide-y" data-testid="event-types">
         {types.map((type, index) => {
           const embedLink = `${group.profile.slug}/${type.slug}`;
-          const calLink = `${CAL_URL}/${embedLink}`;
+          const calLink = `${
+            orgBranding ? `${orgBranding.slug}.${subdomainSuffix()}` : CAL_URL
```

Comment:
**[Blocker-Org]**
In case of orgBranding being true, the result link doesn't have a URL scheme(https://) and thus this link doesn't work when we click on preview

Reply:
Good catch, fixed.